CHANGELOG.md

```diff
@@ -5582,6 +5149,8 @@ Update styles and spacing on the critical error page  ([#20350](https://github.c
 
 [Unreleased]: https://github.com/MetaMask/metamask-extension/compare/v12.5.0...HEAD
 [12.5.0]: https://github.com/MetaMask/metamask-extension/compare/v12.4.0...v12.5.0
+[Unreleased]: https://github.com/MetaMask/metamask-extension/compare/v12.4.1...HEAD
```

Comment:
Also, there is a changelog error here

Reply:
I have noted the instruction you have provided in the meeting, especially regarding resolving the changelog by running the command `yarn lint:changelog` and manually checking the file.